etic_eats_Front/principalmenu.pseudoscript

```diff
@@ -0,0 +1,20 @@
+importar carrinhoPrincipal como carrinho
```

Comment:
@xSimmba Please change this file to uppercased named and also follow [Pseudoscript instructions ](https://github.com/jorgealves/pseudoscript?tab=readme-ov-file#como-correr-um-programa-em-pseudoscript)